x-pack/plugins/lens/server/saved_objects.ts

```diff
@@ -22,6 +23,7 @@ export function setupSavedObjects(core: CoreSetup) {
         uiCapabilitiesPath: 'visualize.show',
       }),
     },
+    migrations,
```

Comment:
This line is slightly different from 7.7 because migrations are now registered to specific types